uint8 NONE = 0
uint8 POINTS = 1

uint8 type
uint64 key
geometry_msgs/Pose pose